app/views/user/show.html.erb

```diff
@@ -0,0 +1,237 @@
+
+            <% count = 1 %>
+
+            <div class="profile-content">
+              <h3></h3>
+              <table class="table table-striped">
+                <thead>
+                  <tr class = "header">
+                    <th>Rank </th>
+                    <th class = "header">Tunes</th>
+                    <th class = "header">Takeout</th>
+                  </tr>
+                </thead>
+                <tbody class="table-body">
+                  <% track = [] %>
+
+                  <% @result.each do |suggestion| %>
+                    <% music = Music.find(suggestion["music_type"].capitalize, suggestion["music_id"]) %>
+                    <% food = Food.yelp_find_by_business(suggestion["food_id"]) %>
+                    <% album_picture = music.images if music.type == "album" %>
+                    <% album_picture = music.images if music.type == "artist" %>
+                    <% album_picture = music.album.images if music.type == "track" %>
+                    <% @id_value = suggestion["music_id"]%>
+
+                    <tr>
+                      <td class="ranking"><%= count %> </td>
+                      <td><img id="<%=suggestion["music_id"]%>" onclick="changeImage('<%=@id_value%>')" src="http://thumb7.shutterstock.com/display_pic_with_logo/1280533/200698931/stock-vector-most-listened-favorite-icon-200698931.jpg" value = "" width="60%" ></td>
+                      <% count += 1 %>
+                      <% if music.type == "album" %> <td><iframe src="https://embed.spotify.com/?uri=spotify%3AalbumA%<%=music.id%>&theme=white&view=coverart" width="300" height="80" frameborder="0" allowtransparency="true"></iframe></td>
+                        <% elsif music.type == "track" %><td><iframe src="https://embed.spotify.com/?uri=spotify:track:<%=music.id%>" width="300" height="80" frameborder="0" allowtransparency="true"></iframe></td>
+                          <% track << music.id %>
+                        <% else %><td class="artist"><iframe src="https://embed.spotify.com/follow/1/?uri=spotify:artist:<%=music.id%>&size=detail&theme=dark" width="300" height="80" scrolling="no" frameborder="0" style="border:none; overflow:hidden;" allowtransparency="true"></iframe></td>
+                      <% end %>
+                    </td>
+                    <td><img src="<%= food.image_url %>"></a></td>
+                    <% raise %>
+
+                      <td>
+
+                        <div class=
+                        <div style="display:block;overflow:hidden;width:500px;height:80px;">
+                        <iframe style="margin-top:-114px;margin-left:-10px" height="200px" scrolling="no"
+                        src="<%= food.url%>" width="375px"></iframe></div>
+                      </td>
+
+
+                    </tr>
+                  <% end %>
+
+                  <script>
+                  function changeImage(id) {
```

Comment:
Wow! Proactive use of javascript! These scripts should be separated into javascript files though!